k8s/deployments.go

```diff
@@ -378,7 +378,11 @@ func newWelcomeContainer(app *models.App) (c *api.Container) {
 func newSlugRunnerContainer(app *models.App, slug string, storageType string) (c *api.Container) {
 	// creating runner container
 	c = newContainer(*app.Name, slugRunnerImage)
-	c.Args = []string{"start", "web"}
+	processType := processTypeWeb
+	if app.ProcessType != nil {
+		processType = *app.ProcessType
+	}
+	c.Args = []string{"start", processType}
```

Comment:
I'm changing my mind to remove this check and force the editing otherwise this check will stay forever.